service/src/test/java/org/cbioportal/service/impl/GenericAssayEnrichmentServiceImplTest.java

```diff
@@ -0,0 +1,261 @@
+package org.cbioportal.service.impl;
+
+import org.cbioportal.model.*;
```

Comment:
Please remove the * import. Sorry, I caught another one.

Reply:
Sorry I noticed the ExpressionEnrichmentTest has the * import, so I just left it here aligned with them.Now I find other test files avoid import *. I have pushed modified version. Shall I modify ExpressionEnrichmentTest as well?